server/service/src/invoice/inbound_shipment/update/mod.rs

```diff
@@ -864,6 +864,9 @@ mod test {
             .find_one_by_id(&stock_line_id)
             .unwrap();
 
+        // Ensure delivered time not updated by status change to verified
+        assert_eq!(invoice.delivered_datetime, None);
```

Comment:
would this test fail now ?

Reply:
Oh good spotting - no, because the dummy data doesn't have the delivered_datetime set as it should 😅

Reply:
Fixed: [6162be9](https://github.com/msupply-foundation/open-msupply/pull/3876/commits/6162be968f7e9598dbf85d1777309697b3d1c840)